app/main/data_requests.py

```diff
@@ -31,3 +33,18 @@ def post_ingest(file: FileStorage, data: dict = None) -> Response:
     else:
         current_app.logger.error(f"Bad response: {request_url} returned {response.status_code}")
         return abort(500)
+
+
+def calculate_days_remaining(current_date=datetime.date(datetime.now())):
+    """Calculate the number of days remaining until a specified submission deadline.
+    The due_date is a str representation of submission deadline in format dd/mm/yyyy.
+    It is set in main/config/envs/default.py
+
+    :param current_date: datetime object representing today's date without timestamp
```

Comment:
could you be more explicit, and state the exact format it's required in?

Reply:
also, the default param doesn't match this, it's a full datetime (ie with the timestamp part too). That makes the test inconsistent with the param. It's not a major problem, as the function is intended to be used with it's default, which is a full datetime object. I guess either

- change the default
- change the value you are testing it with to be a full datetime object, and update the function accordingly

If you don't do this it makes the function harder to re-use

Reply:
If I print the value of datetime.date(datetime.now()) it produces just the date stamp in the format 2023-09-22, do you mean if I change the default parameter to datetime.now()?

Reply:
It's ok, I think this clarified by the comments. But yeah, change the default to `datetime.now().date()` as Cyrus suggested

Reply:
I don't think this has been implemented (same as above)